### ISSUE TITLE
[REQUEST]: Troubleshooting guide for agentless agents

### ISSUE THREAD

**Issue by jen-huang** [elastic/ingest-docs#1466]
### Description

Troubleshooting guide for agentless agents that can be linked from the status info work done in https://github.com/elastic/ingest-dev/issues/3933.

### Resources

TBD

### Collaboration

TBD. The docs and product team will work together to determine the best path forward.

### Point of contact.

**Main contact:** TBD

**Stakeholders:** TBD

**Comment by jen-huang** [comment]
@smriti0321 I heard you are working with @benironside about overall product documentation for agentless. Should I move this issue elsewhere?

**Comment by benironside** [comment]
This is addressed in: https://github.com/elastic/security-docs/pull/6184 
So as far as I'm concerned this issue can be closed.

**Comment by jen-huang** [comment]
@benironside Thanks for letting me know about those docs! The work which needs to link to troubleshooting docs is generic for all agentless integrations, not just CSPM/security ones. For example it will be linked to from Elastic connector integrations too. IMO there should be a general place for docs for agentless integrations that links out to Solution-specific docs as needed.

@nimarezainia @kilfoyle Do we plan to add general agentless docs to existing Fleet and Integrations docs?

**Comment by kilfoyle** [comment]
@jen-huang  I'm sadly not at all up to speed on the agentless work. I can add a page identical to the one Ben created [here](https://security-docs_bk_6184.docs-preview.app.elstc.co/guide/en/serverless/main/agentless-integration-troubleshooting.html) into the Fleet docs, if that makes sense. Would that suffice or do you think there's more required for the generic content?

**Comment by jen-huang** [comment]
@kilfoyle I think that would be a good start. I didn't diff the [ESS](https://security-docs_bk_6184.docs-preview.app.elstc.co/guide/en/security/master/agentless-integrations.html) vs [Serverless](https://security-docs_bk_6184.docs-preview.app.elstc.co/guide/en/serverless/main/agentless-integration-troubleshooting.html) docs so not sure if there is differing info that could be combined or split into multiple sections on the same page.

**Comment by nimarezainia** [comment]
@jen-huang I agree with the fact that we would need a generic troubleshooting guide to deal with agentless infrastructure issues. We don;t really have anything at the moment roadmap wise to deal with this.

I'll discuss with @kpollich and @cmacknz as to what this type of a guide needs to include. At the moment CSPM team owns and operates the agentless infra.

**Comment by bmorelli25** [comment]
This is an older issue. Closing in favor of https://github.com/elastic/ingest-docs/issues/1887 as the updates there will include FAQs
